internal/cmd/operator-sdk/generate/kustomize/manifests.go

```diff
@@ -174,10 +175,12 @@ func (c manifestsCmd) run(cfg *config.Config) error {
 		gencsv.WithBase(c.inputDir, c.apisDir, c.interactiveLevel),
 		gencsv.WithBaseWriter(c.outputDir),
 	}
-	if err := csvGen.Generate(cfg, opts...); err != nil {
+	/*if err := csvGen.Generate(opts...); err != nil {
+		return fmt.Errorf("error generating kustomize bases: %v", err)
+	}*/
+	if err := csvGen.GenerateWithConfig(cfg, opts...); err != nil {
```

Comment:
How about something like
```suggestion
	basePath := filepath.Join(c.outputDir, "bases", cfg.ProjectName + ".clusterserviceversion.yaml"
	base := bases.ClusterServiceVersion{}
	base.BasePath = basePath
	base.OperatorName = c.projectName
	base.OperatorType = projutil.PluginKeyToOperatorType(cfg.Layout)
	base.APIsDir = c.apisDir,
	base.Interactive = isInteractive(c.interactiveLevel) // Move here from CSV generator
	base.GVKs = getGVKs(cfg.Resources) // Implement using loop from makeBaseCreator()
	csv, err := base.GetBase()
	if err != nil {
```